src/components/Tasks/Toolbar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import * as EteSync from 'etesync';
+
+import Switch from '@material-ui/core/Switch';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import IconButton from '@material-ui/core/IconButton';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import QuickAdd from './QuickAdd';
+
+import { PimType } from '../../pim-types';
+
+interface PropsType {
+  defaultCollection: EteSync.CollectionInfo;
+  onItemSave: (item: PimType, journalUid: string, originalItem?: PimType) => Promise<void>;
+  showCompleted: boolean;
+  setShowCompleted: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export default function Toolbar(props: PropsType) {
+  const { defaultCollection, onItemSave, showCompleted, setShowCompleted } = props;
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
```

Comment:
You are probably looking for https://reactjs.org/docs/refs-and-the-dom.html#creating-refs rather than manually setting the ref like this. Though I see that's the recommended way for MUI, so I guess keep it this way. Though :nauseated_face:

Reply:
Yeah I just took it directly off their docs 😬